sup3r/preprocessing/data_handling.py

```diff
@@ -1308,8 +1308,7 @@ def run_all_data_init(self):
         logger.debug(f'Loading data for raster of shape {self.grid_shape}')
 
         # get the file-native time index without pruning
-        time_index = self.raw_time_index
-        n_steps = len(time_index[self.temporal_slice])
+        n_steps = len(self.raw_time_index[self.temporal_slice])
```

Comment:
people think software engineering is super complex but it's mostly creating and then fixing stupid stuff like this

Reply:
lol yeah